units/task_monitor_execution_timeout.go

```diff
@@ -57,12 +54,18 @@ func makeTaskExecutionTimeoutMonitorJob() *taskExecutionTimeoutJob {
 	return j
 }
 
-func NewTaskExecutionMonitorJob(taskID string, execution int, attempt int, ts string) amboy.Job {
+// NewTaskExecutionMonitorJob returns a job to check if a running task has to
```

Comment:
I think you may have meant something like "if a running task has failed to send a heartbeat"